input/otlp/metadata.go

```diff
@@ -185,17 +185,21 @@ func translateResourceMetadata(resource pcommon.Resource, out *modelpb.APMEvent)
 				out.Host = modelpb.HostFromVTPool()
 			}
 			out.Host.Architecture = truncate(v.Str())
+			// TODO: Don't think this is being tested.
```

Comment:
Is this for "host.arch"? I think this is covered by "host" in metadata_test, but feel free to add a test if it is not covered.

Reply:
You're right. It wasn't obvious to me at first glance. So I made a note for me to get back too, and/or for someone to comment in the Draft, like you just did. Thank you. Also, in the future I'll use FIXME instead to be more clear.